Add missing symbol for WebCore::TextIterator::getLocationAndLengthFromRange

Signed-off-by: Markos Chandras <markos.chandras@imgtec.com>

From f5bb58f9096061f00c948e668335811d364ba360 Mon Sep 17 00:00:00 2001
From: "kov@webkit.org" <kov@webkit.org@268f45cc-cd09-0410-ab3c-d52691b4dbfc>
Date: Thu, 7 Feb 2013 13:48:46 +0000
Subject: [PATCH] Unreviewed build fix after r141196 for 32 bits autotools.

* Source/autotools/symbols.filter: restore 32 bits version of the
WebCore::TextIterator::getLocationAndLengthFromRange(WebCore::Node*,
WebCore::Range const*, unsigned int&, unsigned int&) symbol.

git-svn-id: http://svn.webkit.org/repository/webkit/trunk@142105 268f45cc-cd09-0410-ab3c-d52691b4dbfc
---
 ChangeLog                       | 8 ++++++++
 Source/autotools/symbols.filter | 1 +
 2 files changed, 9 insertions(+)

diff --git a/Source/autotools/symbols.filter b/Source/autotools/symbols.filter
index 9d730b3..2edefaf 100644
--- a/Source/autotools/symbols.filter
+++ b/Source/autotools/symbols.filter
@@ -57,6 +57,7 @@ _ZN7WebCore11HistoryItem16setDocumentStateERKN3WTF6VectorINS1_6StringELj0EEE;
 _ZN7WebCore11HistoryItem16setDocumentStateERKN3WTF6VectorINS1_6StringELm0EEE;
 _ZN7WebCore11MemoryCache14resourceForURLERKNS_4KURLE;
 _ZN7WebCore12TextIterator26rangeFromLocationAndLengthEPNS_13ContainerNodeEiib;
+_ZN7WebCore12TextIterator29getLocationAndLengthFromRangeEPNS_4NodeEPKNS_5RangeERjS6_;
 _ZN7WebCore12TextIterator29getLocationAndLengthFromRangeEPNS_4NodeEPKNS_5RangeERmS6_;
 _ZN7WebCore12PrintContext20pageNumberForElementEPNS_7ElementERKNS_9FloatSizeE;
 _ZN7WebCore13createWrapperEPN3JSC9ExecStateEPNS_17JSDOMGlobalObjectEPNS_4NodeE;
-- 
1.8.3.2

